lib/playground.dart

```diff
@@ -315,21 +341,47 @@ class Playground {
     if (context.focusedEditor == 'dart') {
       ga.sendEvent('main', 'help');
 
+      var input;
       Position pos = editor.document.cursor;
-      var input = new SourceRequest()
+      int offset = editor.document.indexFromPos(pos);
+
+      if (querySelector(".CodeMirror-hint-active") != null) {
```

Comment:
I would say, whichever object logically contains the UI for the doc panel. Probably `Playground`.